_posts/2016/05/2016-05-11-first-tertile.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+authors: ["Raniere Silva"]
+title: "First bimonthly report from 2016 Steering Committee"
+date: 2016-05-11
+time: "10:00:00"
+category: ["Community"]
+---
+On May 8th and 9th the Steering Committee had an in-person meeting at Cold
+Spring Harbour Laboratory to conclude the discussions during the first
+two months and make plans for the next ten months of activities.
+In the following weeks we will write more about the results of the in-person
+meeting on the blog but for now we want to give you an overview.
+
+## Partnerships
+
+Our Executive Director is doing an amazing job increasing the number of Software
+Carpentry [partners](http://software-carpentry.org/scf/partners/). Last year,
+we were asked to provide a join partnership agreement with [Data
+Carpentry](http://datacarpentry.org/)
+and after months of work this agreement is now
+[available](http://software-carpentry.org/scf/join/).
+
+## Instructor Training
+
+Last year we tried several models for Instructor Training
+that now are more stable and has an [write
+version](http://swcarpentry.github.io/instructor-training/)
```

Comment:
“write version” → “written version”.  And maybe kick off a new sentence: “… Training.  Now the model has stabilized and has a written version, thanks to …”.